packages/app/src/background/services/credentials/index.ts

```diff
@@ -146,8 +237,17 @@ export default class VerifiableCredentialsService implements IBackupable {
         type: "basic",
       },
     });
-
-    return true;
+    const tabs = await browser.tabs.query({ active: true });
+    await Promise.all(
+      tabs.map((tab) =>
+        browser.tabs
+          .sendMessage(tab.id!, {
+            action: "addVerifiableCredential",
```

Comment:
This is magic string. The code should be the same as for status change.

Reply:
Makes sense. I think we can refactor the [`EventName` type](https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/3384e79006a4aef46c698e083c2b29ad5eee9782/packages/providers/src/event/types.ts#L16) into an `Enum`. I also think it makes sense to separate this change into a different PR since it affects many event handling files and this change is already getting quite large.